docs/guides/distribution/app-stores.md

```diff
@@ -0,0 +1,86 @@
+# Publishing Tauri Applications to App Stores
+
+In this guide, we will cover the process of publishing Tauri applications on various app stores, including Snapstore (Linux), Flathub (Linux), Chocolatey (Windows), and AppImage (Linux). Please note that the macOS platform is not covered due to the unavailability of a testing device.
+
+## Snapstore
+
+**Note**: This process requires a Linux operating system, and WSL may not be suitable depending on the application type.
+
+### Method 1: Utilizing an External npm Library
+
+1. Begin by creating a `.snap` package for your application using the [tauri-snap-packager](https://www.npmjs.com/package/tauri-snap-packager) npm package.
```

Comment:
tbh i hoped we could have get https://github.com/tauri-apps/tauri/pull/6532 in in time because i'd prefer to have most of the bundler tooling in our control, especially if the nieche community project hasn't received an update in over 2 years.
I don't wanna talk down on that project of course but i'm not sure it quite aligns with our pretty strict views on security etc.

But still, not quite sure how to proceed so let's leave it for now i guess.

Reply:
I know. In my case that method didn't even work, I had to do it by hand because I used WSL instead of an actual Linux distribution. As you mentioned earlier, this and many other issues related to it could be solved with a new version of the Tauri build tool. Unfortunately, I don't know how to help you on that because I feel like that could become very complex.